src/components/Breadcrumbs/index.tsx

```diff
@@ -65,7 +70,7 @@ const Breadcrumbs = ({
     .slice(startDepth)
 
   return (
-    <Breadcrumb {...props}>
+    <Breadcrumb {...props} dir={dir}>
```

Comment:
Most instances of the `dir` prop should now be removed as this is redundant and is taken care of under the hood in the Chakra component structure/theme or by the use of an `inset*` prop.

An exception is where flexbox is used for content positioning, where CSS is not smart enough to handle RTL automatically with flex props. (I will point out the exceptions)

Reply:
See other comment about specifically why I'm overriding these at the component level... same applied to this one.